src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -618,6 +625,16 @@ var adyenHelperObj = {
     return format;
   },
 
+  getDivisorForCurrency(amount) {
+    let fractionDigits = adyenHelperObj.getFractionDigits(amount.currencyCode);
+    let divideBy = 1;
```

Comment:
I think we can make this easier, for example we can use `divideBy = Math.pow(10, fractionDigits)`